docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,302 @@
+
+1. [Install Kyma](../../04-operation-guides/operations/01-install-kyma.md) 2.0 or higher.
+
+2. Create a Namespace. Run:
+
+   ```bash
+   kubectl create ns {NAMESPACE_NAME}
+   ```
+
+3. Create a Secret containing credentials for your DNS cloud service provider account. See the [official External DNS Management docuemntation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Remember to edit the **metadata.namespace** parameter and provide your {NAMESPACE_NAME} as the required value.  Once the YAML file with the relevant parameters is ready, create a Secret Custom Resource (CR). Run the following command:
+
+   ```bash
+   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+   ```
+
+4. Set up the `external-dns-management` component.
+
+- Create a DNSProvider CR. Export values of the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters as environment variables. As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE_NAME}
+   export SPEC_TYPE={PROVIDER_TYPE}
+   export SECRET={SECRET_NAME}
+   export DOMAIN={CLUSTER_DOMAIN}
+   ```
+  
+  Create a DNSProvider CR. Run:
+
+  <div tabs>
+  <details>
+  <summary>
+  Gardener cluster
+  </summary>
+
+  ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: $NAMESPACE
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     type: $SPEC_TYPE
+     secretRef:
+       name: $SECRET
+     domains:
+       include:
+         - $DOMAIN
+   EOF  
+   ```
+
+  </details>
+  <details>
+  <summary>
+  Non-Gardener cluster
+  </summary>
+
+  ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: $NAMESPACE
+   spec:
+     type: $SPEC_TYPE
+     secretRef:
+       name: $SECRET
+     domains:
+       include:
+         - $DOMAIN
+   EOF  
+   ```
+
+   </details>
+   </div>
+
+- Create a DNSEntry CR. Export values of the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters as environment variables. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE_NAME}
+   export SUBDOMAIN={YOUR_SUBDOMAIN}
+   export IP={SECRET_NAME}
```

Comment:
Ehm, it should be external LoadBalancer IP, not the secret name. There's a Note below that shows how to fetch this IP.